src/xcode/ENA/ENA/Source/Services/Exposure Submission/ExposureSubmissionService+Protocol.swift

```diff
@@ -29,6 +29,8 @@ protocol ExposureSubmissionService: class {
 	var devicePairingConsentAcceptTimestamp: Int64? { get }
 	var devicePairingSuccessfulTimestamp: Int64? { get }
 	
+	var isSubmissionConsentGiven: Bool { get set }
```

Comment:
Even with these great names it's nice to see a comment on these properties.

```suggestion
      /// Indicates wether the user allowed to submit test results automatically to the federation gateway or not. Defaults to `false`.
	var isSubmissionConsentGiven: Bool { get set }
```

Comments ftw 🤘

Reply:
Comments ftw 🤘 👍